rust/operator-binary/src/operations/pdb.rs

```diff
@@ -0,0 +1,107 @@
+use std::cmp::max;
+
+use snafu::{ResultExt, Snafu};
+use stackable_operator::{
+    builder::pdb::PodDisruptionBudgetBuilder, client::Client, cluster_resources::ClusterResources,
+    commons::pdb::PdbConfig, kube::ResourceExt,
+};
+use stackable_zookeeper_crd::{ZookeeperCluster, ZookeeperRole, APP_NAME, OPERATOR_NAME};
+
+use crate::zk_controller::ZK_CONTROLLER_NAME;
+
+#[derive(Snafu, Debug)]
+pub enum Error {
+    #[snafu(display("Cannot create PodDisruptionBudget for role [{role}]"))]
+    CreatePdb {
+        source: stackable_operator::error::Error,
+        role: String,
+    },
+    #[snafu(display("Cannot apply role group PodDisruptionBudget [{name}]"))]
```

Comment:
Not role-group right?
```suggestion
    #[snafu(display("Cannot apply PodDisruptionBudget [{name}]"))]
```

Reply:
yes